README.md

```diff
@@ -1,21 +1,68 @@
-      * [Build the app](#build-the-app)
-    * [Windows](#windows)
-  * [Downloads](#downloads)
-    * [macOS](#macos)
-    * [Windows](#windows)
-    * [Linux (64 bit only)](#linux-64-bit-only)
-  * [Change log](#change-log)
-  * [Documentation](#documentation)
-  * [Contribute](#contribute)
-
-# Build instructions
+<h1 align="center">
+  <a href="https://toggl.com"><img src="https://toggl.com/site/images/media-toolkit/logo_02-644bd26148b73c19d9c91e5baecd8e31.jpg" alt="Toggl" width="200"></a>
+</h1>
+
+<h4 align="center">Native desktop applications for the leading time tracking tool <a href="https://toggl.com" target="_blank">Toggl</a>.</h4>
+
+<p align="center">
+    <a href="https://github.com/toggl/toggldesktop/commits/master">
+    <img src="https://img.shields.io/github/last-commit/toggl/toggldesktop.svg?style=flat&logo=github&logoColor=white"
+         alt="GitHub last commit">
+    <a href="https://github.com/toggl/toggldesktop/issues">
+    <img src="https://img.shields.io/github/issues-raw/toggl/toggldesktop.svg?style=flat&logo=github&logoColor=white"
+         alt="GitHub issues">
+    <a href="https://github.com/toggl/toggldesktop/pulls">
+    <img src="https://img.shields.io/github/issues-pr-raw/toggl/toggldesktop.svg?style=flat&logo=github&logoColor=white"
+         alt="GitHub pull requests">
+    <img src="https://img.shields.io/badge/licence-BSD--3-green"
+         alt="Licence BSD-3">
+</p>
+
+<p align="center">
+  <a href="#about">About</a> •
+  <a href="#download">Download</a> •
+  <a href="#build">Build</a> •
+  <a href="#change-log">Change log</a> •
+  <a href="#contribute">Contribute</a>
+</p>
+
+# About
+
+  **Toggl Desktop** is a Toggl time tracking client with many helper functions that make tracking time more effortless and smooth. Features such as Idle detection, reminders to track and Pomodoro Timer make this app a great companion when productivity and efficiency is the goal.
+
+<img src="https://user-images.githubusercontent.com/842229/63856838-3a869580-c9ab-11e9-9e36-7db23059ce29.png"
+         alt="Toggl Desktop apps">
+
+# Download
+
+Toggl built and signed apps for all platforms
+
+## Mac
+
+<br>
+<a href="https://toggl.github.io/toggldesktop/download/macos-stable/">64bit dmg</a>&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href='https://itunes.apple.com/ee/app/toggl-desktop/id957734279?mt=12'>
+  Mac App Store</a>
+<br/>
+<br/>
+<i>Officially macOS 10.11 and newer stable macOS versions are supported.</i>
+
+## Windows
+
+<br/>
+<a href="https://toggl.github.io/toggldesktop/download/windows-stable/">32Bit installer</a>&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://toggl.github.io/toggldesktop/download/windows64-stable/">64Bit installer</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://chocolatey.org/packages/toggl">Chocolatey</a>&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;<a href='//www.microsoft.com/store/apps/9nk3rf9nbjnp?cid=storebadge&ocid=badge'>Microsoft Store</a>
```

Comment:
I'll put a 64bit link before a 32bit one and fix the capitalization of `bit` for consistency reasons.